packages/standalone-uis/execution-timeline/webpack.config.js

```diff
@@ -35,11 +35,11 @@ module.exports = {
         test: /(?:\.ngfactory\.js|\.ngstyle\.js|\.ts)$/,
         use: '@ngtools/webpack',
       },
-      {
-        test: /\.js$/,
-        use: ['source-map-loader'],
-        enforce: 'pre',
-      },
+      // {
+      //   test: /\.js$/,
+      //   use: ['source-map-loader'],
+      //   enforce: 'pre',
+      // },
```

Comment:
Is there a reason why this is commented out instead of removed?

Reply:
This was causing some build warnings. I think I'll revert back and tackle as part of a different PR.

Reply:
@andrewthauer I didn't have a chance to investigate this, but is there any reason the execution timeline needed to be ejected? Could we have gotten away with keeping it to using the cli by chance?

Reply:
I'm not 100% sure, but I think this was done because it's treated as a library and consumed by the `performance-profiler-plugin`. However, something like ng-packager would probably help and potentially allow for using the standard CLI. Some additional research should be required. I've actually I've recently considering looking at `rollup` vs `webpack` because it makes it easier to have multiple output targets in a single build (commonjs, umd, esm, etc.).